### DISCUSSION TITLE
Setting/Editting Information from the inverse side not working

### DISCUSSION THREAD

**Discussion by osviel91** [symfony/symfony#46504] [Q&A]
Hi, i'm having a issue when i'm trying to add/edit a Entity from the inverse side of a relationship.

Here is my case: I have a Objetive and a Block of Objetives (Block). The owning side is Block. The relationship should be A Block have many Objetives. 

So from the Objetive side i can edit/update the Block related but from the Block side i can't edit or update the Objetives that are related. Simply it's ignored the action and no error showed up.

this is the way i'm relating it in the Objetive Class:
<pre>
#[ORM\ManyToOne(targetEntity: Block::class, inversedBy: 'objetives')]
private $block;
</pre>

this is the way i'm relating it into the Block class:
<pre>
#[ORM\OneToMany(mappedBy: 'block', targetEntity: Objetive::class)]
private $objetives;
</pre>

The add method it's the one is autogenerated for the make:entity command...

<pre>
public function addObjetive(Objetive $objetive): self
    {
        if (!$this->objetive->contains($objetive)) {
            $this->objetives[] = $objetive;
            $objetivo->setBlock($this);
        }
        return $this;
    }
</pre>

**Comment by ghost** [comment]
Did you copy past this `addObjetive` method without modifying? If so then there is a typo in line `$objetivo->setBlock($this);`. However PHP should crash at this line so you would see it.
The problem you described is a common problem. I don't remember the solution now as I don't use this feature anymore but I remember SymfonyCasts explains it in their Doctrine tutorials. Here is an old one https://symfonycasts.com/screencast/symfony2-ep3/doctrine-inverse-relation but I'm pretty confident they talk about that in newer tutorials as well, just do some research on their website. 

BTW. There's no such word as Objetive, I think you mean Obje**c**tive.

**Comment by slaci** [comment]
This is how doctrine works as far as I know. From the [docs](https://www.doctrine-project.org/projects/doctrine-orm/en/2.11/reference/working-with-associations.html#working-with-associations) this part:

> When a bidirectional association is updated, Doctrine only checks on one of both sides for these changes. This is called the [owning side](https://www.doctrine-project.org/projects/doctrine-orm/en/2.11/reference/unitofwork-associations.html) of the association.

When I last encountered this problem, the same happened for me. Doctrine just ignored what was added from the inverse side no matter what I did, but it seems to be a known limitation according to the docs.
